app/views/job_runs/show.html.erb

```diff
@@ -1,4 +1,8 @@
-<%= link_to '[all jobs]',job_runs_path %>
+<div class="container">
+  <dl class="row">
```

Comment:
Does this `<dl>` element make sense here?  This is a standalone template, so it doesn't seem like a "description list" is accurate here, since there are no `<dd>` or `<dt>` elements with it in this context.

Reply:
yup, removed